tests/conftest.py

```diff
@@ -0,0 +1,35 @@
+"""Provide project scoped test fixtures."""
+import signal
+import subprocess
+import time
+import uuid
+
+import pytest
+
+from tests.common import ENGINE_MODULE, TOKEN, URL
+from tests.fake_client import TestClient
+
+
+@pytest.fixture(name="engine")
+def setup_engine():
```

Comment:
Right now this fixture is function scoped, so will be run for each test function. We can change this later, but should then think about consequences of keeping the same engine around for multiple tests.